### DISCUSSION TITLE
Show player homes in Dynmap

### DISCUSSION THREAD

**Discussion by kirin-3** [Zrips/CMI#6646] [Feature Requests]
This is something you can do with essentials, I really wish it was possible with CMI too. Adding warps to Dynmap would be pretty cool too.

**Comment by CloudeLecaw** [comment]
Warps would be cool. Maybe a new setting that is disabled by Default.

For homes...no pls not ;)

**Comment by shantek** [comment]
I'd love to see this too. Real handy to keep an eye on where players are setting their homes and if it's places they don't belong.
